source/english/partials/curriculum/_daily_schedule.html.haml

```diff
@@ -23,44 +27,51 @@
+              %p Eat refuel, gather your energy for the afternoon ahead.
           %tr
-            %td{data:{header: "Time"}} 2.00pm
-            %td{data:{header: "Activity"}} Meditation
+            %td{data:{header: "Time"}} 13.00pm
+            %td{data:{header: "Activity"}} Demo
             %td{data:{header: "Description"}}
-              %p Programming is extremely challenging. Meditation helps keep a lid on things, keeps you grounded, and keeps your brain sharp.
+              %p The students will take part in a video chat, a coach will talk about a tool or coding concept and also show the students how to use the tool or concept in a codebase.
           %tr
-            %td{data:{header: "Time"}} 2.30pm
-            %td{data:{header: "Activity"}} Standup
+            %td{data:{header: "Time"}} 14.00pm
+            %td{data:{header: "Activity"}} Demo
             %td{data:{header: "Description"}}
-              %p Same as the morning. Keep the feel-goods flowing.
+              %p The students will take part in a video chat, a coach will talk about a tool or coding concept and also show the students how to use the tool or concept in a codebase.
           %tr
-            %td{data:{header: "Time"}} 3.00pm
-            %td{data:{header: "Activity"}} Breakout
+            %td{data:{header: "Time"}} 15.00pm
+            %td{data:{header: "Activity"}} Pair Programming
             %td{data:{header: "Description"}}
-              %p Sometimes students don’t need this one. They’re usually pretty keen to get back to the project they were working on.
+              %p Students pair program with each other, working on their weekly project, building real applications using tools and practices that are the standard in modern development environments.
           %tr
-            %td{data:{header: "Time"}} 4.00pm
-            %td{data:{header: "Activity"}} Pair Programming
+            %td{data:{header: "Time"}} 16.00pm
+            %td{data:{header: "Activity"}} Code Katas 
             %td{data:{header: "Description"}}
-              %p More pair programming on our challenge-based curriculum, with coach and alumni helper support.
+              %p We end the day with code katas, katas are the best way to kepp your coding skills up.
           %tr
-            %td{data:{header: "Time"}} 8.30pm
-            %td{data:{header: "Activity"}} Socialising
+            %td{data:{header: "Time"}} 17.00pm
+            %td{data:{header: "Activity"}} Day ends
             %td{data:{header: "Description"}}
-              %p A pecha kucha night, a trip to a software craftsmanship meetup at Yammer, a hosted event, or a film night...we love to organise stuff to keep your social bar topped up.
-    .button-row.small-padding-top
-      %p
-        = link_to "Read more about our Coaching style", "https://blog.makersacademy.com/teacher-time/", target: "_blank", class: "button button--horizontal"
+              %p This is the coaches end of the day not the students.
```

Comment:
During the evening you will work with fellow students on your group assignments and projects. The coaches will see your progress next day